ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -645,6 +645,206 @@ public final class AliasTokens {
         }
     }
 
+    // MARK: Shared Colors
+
+    public enum SharedColorsTokens: CaseIterable {
+        // Red
+        case redBackground1
+        case redBackground2
+        case redBackground3
+        case redForeground1
+        case redForeground2
+        case redForeground3
+        case redBorderActive
+        case redBorder1
+        case redBorder2
+        // Dark orange
+        case darkOrangeBackground1
+        case darkOrangeBackground2
+        case darkOrangeBackground3
+        case darkOrangeForeground1
+        case darkOrangeForeground2
+        case darkOrangeForeground3
+        case darkOrangeBorderActive
+        case darkOrangeBorder1
+        case darkOrangeBorder2
+        // Yellow
+        case yellowBackground1
+        case yellowBackground2
+        case yellowBackground3
+        case yellowForeground1
+        case yellowForeground2
+        case yellowForeground3
+        case yellowBorderActive
+        case yellowBorder1
+        case yellowBorder2
+        // Green
+        case greenBackground1
+        case greenBackground2
+        case greenBackground3
+        case greenForeground1
+        case greenForeground2
+        case greenForeground3
+        case greenBorderActive
+        case greenBorder1
+        case greenBorder2
+        // Berry
+        case berryBackground1
```

Comment:
talked offline: if this isn't used outside of OOF badge, it should not be an alias token